src/moonlink_connectors/src/pg_replicate/conversions/text.rs

```diff
@@ -351,13 +354,17 @@ impl TextFormatConverter {
                         }
                         Kind::Array(_) => {
                             // TODO: Multi-dimensional arrays not yet implemented
-                            // TODO: Need to print out the unsupported type (included in error type)
-                            Err(FromTextError::InvalidConversion())
+                            Err(FromTextError::InvalidConversion(format!(
+                                "multi-dimensional arrays"
```

Comment:
```suggestion
                                format!("{:?}", typ)
```
so we could know which type it is, multi-dim array doesn't help too much